federation/backends/collections.md

```diff
@@ -1,6 +1,27 @@
 # Collections
 This page describes requirements on STAC collection metadata for backend providers in the openEO platform federation. These requirements should be considered an addition to what is already required by the [openEO API](https://api.openeo.org/#tag/EO-Data-Discovery/operation/describe-collection).
 
+## Collection availability
+
+The collections are a key asset of the platform, and users need to know what they can expect from a certain collection.
+The federation relies on openEO collection metadata following the STAC specification to communicate this towards the user. Backend providers
+need to comply with these requirements for their collection:
+
+The 'experimental' flag from https://stac-extensions.github.io/version/v1.1.0/schema.json needs to be set on experimental collections. 
+A collection can be experimental either because there are issues with the actual data or catalog, or because of backend specific issues that make the use of the collection unstable. 
+A collection marked as experimental does not need to comply with further requirements. 
+
+Collections need to indicate the key 'providers' that are responsible for ensuring access to the data and continuity in the case of active missions. The user may depend on the guarantees offered by these
+providers with respect to the properties (timeliness, completeness,...) of a specific collection. 
+The providers with role 'host' and 'producer' are mandatory.
+
+The collection description needs to specify known limitations with respect to the original collection. For instance, if only a subset of the full archive is available, this should be indicated.
+
+Collections without an end time are assumed to be active missions. By default, 99% of items in these collections should be available within 48 hours after being published by the producer. This gives users a basic guarantee with respect to timeliness of products. 
+
+
```

Comment:
Do we want to include here the min. notice time a backend has to give to the user in case a non experimental collection will be removed or is this to much detail and we keep it for the OLAs (might be different per backend but a minimum agreement 6 months? can maybe be included here).

Reply:
yes, that's a good idea, will add in next update.

Reply:
done